<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UniversalSettingUi</class>
 <widget class="QWidget" name="UniversalSettingUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>806</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <italic>false</italic>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>UniversalSettingUi</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="WG_universal" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="WG_theme" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QtMaterialFlatButton" name="btn_color_1">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QtMaterialFlatButton" name="btn_color_2">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QtMaterialFlatButton" name="btn_color_3">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QtMaterialFlatButton" name="btn_color_4">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QtMaterialFlatButton" name="btn_color_5">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QtMaterialFlatButton" name="btn_color_6">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>#</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QtMaterialTextField" name="LE_color">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="LB_chinese">
           <property name="text">
            <string>中文字体：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QFontComboBox" name="FCB_chinese_font"/>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="SB_chinese_number">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>30</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QtMaterialCheckable" name="CB_chinese_isbold">
           <property name="text">
            <string>粗体</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="LB_english">
           <property name="text">
            <string>英文字体：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QFontComboBox" name="FCB_english_font"/>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="SB_english_number">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>30</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QtMaterialCheckable" name="CB_english_isbold">
           <property name="text">
            <string>粗体</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtMaterialFlatButton</class>
   <extends>QPushButton</extends>
   <header location="global">qtmaterialflatbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialTextField</class>
   <extends>QLineEdit</extends>
   <header>qtmaterialtextfield.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialCheckable</class>
   <extends>QCheckBox</extends>
   <header location="global">lib/qtmaterialcheckable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
